src/reticulum.js

```diff
@@ -47,18 +46,22 @@ class ReticulumChannel extends EventEmitter {
         }
         return;
       }
-      // this guy was not previously present, notify for a join
+
+      // this user was not previously present, notify for a join
       this.emit('join', Date.now(), id, mostRecent.presence, mostRecent.profile.displayName);
     });
 
     this.presence.onLeave((id, curr, p) => {
-      if (BOT_SESSION_IDS.has(id)) {
+      const mostRecent = p.metas[p.metas.length - 1];
+
+      if (mostRecent.context.discord) {
```

Comment:
I wish we had typescript, will context ever be undefined?

Reply:
Hmm, yeah Idk. On second glance it looks like Hubs does check for it being undefined so it probably should here as well https://github.com/mozilla/hubs/blob/master/src/react-components/presence-list.js#L111